windrose/windrose.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import pandas as pd
+
+
+class WindroseAnalysis:
+    def __init__(self, ref_n=90, ref_e=0):
+        self.north = ref_n
+        self.east = ref_e
+
+    @staticmethod
```

Comment:
What is the reason for defining this as a staticmethod?

Reply:
I have looked into this. There was an error showing previously that the calculate_wind_frequency function is static, so adding the staticmethod decorator does not need to use any information from the class it is in. It is just working with the inputs given to it. Now I have changed the code and removed the decorator and added the self function.